### ISSUE TITLE
Luminosity contrast ratio for border lines of controls is less than 3:1 present under Style editor.

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#81]
**User Experience:**
Low vision user would face difficulty to see ‘Input fields, Icon controls and Scroll bar’ if luminosity ratio is less than 3:1.

**Repro Steps:**
1. Open Azure portal with valid credentials and create a API Management.
2. Click Developer Portal button to navigate to the portal.
3. Wait portal to load in design mode.
4. Navigate to the Left nav and select "Styles" button.
5. Navigate to style page and select Heading Level 1.
6. Navigate to the various controls of the "Heading Level 1" dialog box.
7. Open Color contrast analyser (CCA) tool and check luminosity contrast ratio of border lines.
8. Verify if luminosity ratio for the border lines of Input fields, Icon controls and Scroll bar.

**Actual Result:**
Luminosity contrast ratio for border lines of Input fields, Icon controls and Scroll bar is less than 3:1.

**Expected result:**
Luminosity contrast ratio should be greater than or equal to 3:1 for border lines of Input fields, Icon controls and Scroll bar, present under ‘Heading level 1’ popup. At present it is 1.6:1.

**Note:**
Similar issue exist with Chrome. 

**MAS Reference:**
MAS 1.4.11 - Non-text Contrast ***

**Reference Links**
Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to edad team”.
Please reach out to C&AI Teams channel for any process related queries.

![image](https://user-images.githubusercontent.com/2320302/76659878-c9689300-6534-11ea-8339-590ee7286c04.png)
